src/Source/OrientedImageSource.js

```diff
@@ -0,0 +1,68 @@
+import Source from 'Source/Source';
+import Fetcher from 'Provider/Fetcher';
+import OrientationUtils from 'Utils/OrientationUtils';
+import CameraCalibrationParser from 'Parser/CameraCalibrationParser';
+import GeoJsonParser from 'Parser/GeoJsonParser';
+import Coordinates from 'Core/Geographic/Coordinates';
+
+const coord = new Coordinates('EPSG:4978', 0, 0, 0);
+
+/**
+ * @classdesc OrientedImageSource is a specific source used to load oriented images.
+ * @extends Source
+ */
+class OrientedImageSource extends Source {
+    /**
+     * @constructor
+     * @param { Object } source - Configuration object
+     * @param { Object } source.url - It's composed of 3 urls :
+     * @param { string } source.url.orientations - Url to a GEOJSON file describing feature points. It describre position and orientation of each panoramic.
+     * @param { string } source.url.calibrations - Url of a configuration file for the set of cameras. see [CameraCalibrationParser]{@link module:CameraCalibrationParser.parse}
+     * @param { string } source.url.images - Url for all the textures.
+     * This Url must contains {sensorId} and {cameraId}, and these pattern will be replaced be the {@link OrientedImageProvider} o find the good texture for each camera for each panoramic.
+     */
+    constructor(source) {
+        source.format = source.format || 'json';
+        super(source);
+        const promises = [];
+
+        this.url = source.url;
+        // option for GeoJsonParser
+        this.mergeFeatures = false;
+        this.crsOut = source.crsOut;
```

Comment:
` this.crsOut` should not exist it should use `layer.projection` instead. `layer.projection` is the destination projection.